networking/hardware_networks/using-pod-level-bonding.adoc

```diff
@@ -0,0 +1,38 @@
+[id="using-pod-level-bonding"]
+= Using pod-level bonding
+include::modules/common-attributes.adoc[]
+:context: using-pod-level-bonding
+
+toc::[]
+
+
+Bonding at the pod level is vital to enable workloads inside pods that require high availability. With pod-level bonding, you can create a bond interface from multiple virtual function interfaces (single root I/O virtualization (SR-IOV) in kernel mode interface). The virtual functions are passed into the pod and attached to a kernel driver.
+
+Scenarios where pod level bonding is required include creating a bond interface from multiple virtual functions on different physical functions. Creating a bond interface from two different interfaces on the host can be used to achieve high availability at pod level.
+
+Note:  Link Aggregation Control Protocol (LACP) mode is not supported, as is the case for Red Hat OpenShift Container Platform.
```

Comment:
I didn't understand this

`as is the case for Red Hat OpenShift Container Platform.`